app/models/setting.rb

```diff
@@ -184,7 +184,12 @@ def defaults
         "remote_census.response.gender": "",
         "remote_census.response.name": "",
         "remote_census.response.surname": "",
-        "remote_census.response.valid": ""
+        "remote_census.response.valid": "",
+        "sdg.process_debates": false,
+        "sdg.process_proposals": false,
+        "sdg.process_polls": false,
+        "sdg.process_investments": false,
```

Comment:
For consistency with the settings starting with `process`, how about `sdg.process.budgets`? And, who knows, maybe in the future budgets or headings will be related to SDGs :thinking:.